src/main/java/org/jenkinsci/plugins/envinject/service/EnvInjectEnvVars.java

```diff
@@ -140,6 +140,7 @@ public int executeScript(boolean loadFromMaster,
             groovyShell.setVariable(entryVariable.getKey(), entryVariable.getValue());
         }
         groovyShell.setVariable("out", logger.getListener().getLogger());
+        groovyShell.setVariable("currentListener", logger.getListener());
```

Comment:
Other Groovy steps use the "listener" variable, but I agree that this PR should follow the legacy approach